### ISSUE TITLE
Low-contrast text in the carousel element on the top page

### ISSUE THREAD

**Issue by bick-jp** [internetarchive/openlibrary#7848]
Low-contrast text occurs between the subtext and the background in all six elements of the carousel at the top of the home page. The subtext color should be changed to a darker color.

### Evidence / Screenshot (if possible)
![Screen Shot 2023-05-05 at 2 11 30 PM](https://user-images.githubusercontent.com/8709562/236537537-e9305d67-b544-4542-b7d8-e1c124d8fb99.png)
![Screen Shot 2023-05-05 at 2 11 34 PM](https://user-images.githubusercontent.com/8709562/236537558-ecc8e3fc-e2aa-440e-8ea6-7fb3f367bc83.png)

### Relevant url?

[https://openlibrary.org/](https://openlibrary.org/)

### Steps to Reproduce

1. Go to [https://openlibrary.org/](https://openlibrary.org/)
2. Use color picker (e.g., [ColorPick Eyedropper](https://chrome.google.com/webstore/detail/colorpick-eyedropper/ohcpnigalekghcmgcdcenkpelffpdolg)) and select the text and background, which will give you #A49EA1 for the text and #FCFBF7 for the background.
3. Put the color codes into [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/)
4. It will give you Contrast Ratio 2.54:1

* Actual: Current contrast ratio is 2.54:1 (Text color is #A49EA1 and Background color is #FCFBF7)
Contrast Ratio 2.54:1

* Expected: The color of the subtext should be darker so that the contrast ratio can be 4.5:1 to comply WCAG 2.0 level AA or 7:1 to comply WCAG 2.0 level AAA

### Details

- **Logged in (Y/N)?** N
- **Browser type/version?** Google Chrome Version 112.0.5615.137 (Official Build) (arm64)
- **Operating system?** macOS Monterey Version 12.3.1
- **Environment (prod/dev/local)?** prod

**Comment by cdrini** [comment]
Note: It looks like Wikimedia is "partially conformant with WCAG 2.1 level AA"

> Wikimedia Foundation Design website is partially conformant with WCAG 2.1 level AA. Partially conformant means that some parts of the content do not fully conform to the accessibility standards.
> https://design.wikimedia.org/accessibility-statement.html

That seems like a reasonable pattern for us to follow :)

**Comment by kmcgonigle** [comment]
@bick-jp, @cdrini and @mekarpeles, I spent a little time playing around with color contrast. Here are a few options we could go with to satisfy the minimum color contrast to pass WCAG AA:

## Option 1: Lighter background, darker text
<img width="1050" alt="Screen Shot 2023-05-24 at 8 12 09 PM" src="https://github.com/internetarchive/openlibrary/assets/13765801/51748a2b-3a77-47a5-baeb-3ed5cfa650b6">


Slightly lighter background: `#FFFEFA`
Slightly darker secondary text: `#7B7476`
[Contrast ratio: 4.52:1](https://webaim.org/resources/contrastchecker/?fcolor=7B7476&bcolor=FFFEFA)

---
## Option 2: All text same color (background unchanged)
<img width="1056" alt="Screen Shot 2023-05-24 at 8 20 24 PM" src="https://github.com/internetarchive/openlibrary/assets/13765801/e0d53792-575f-49f9-b608-6619fb73b64f">


Secondary text color changed to match primary text color: `#333333`
[Contrast ratio: 12.51:1](https://webaim.org/resources/contrastchecker/?fcolor=333333&bcolor=FFFEFA)


Making this change made the text hierarchy a little less clear, so I also introduced just an extra smidge of spacing between primary and secondary text lines.

---
These are both really subtle -- I tried to make the smallest change that would bring us in line with WCAG AA. Let me know what you think.  😊

**Comment by bick-jp** [comment]
Lovely :) I like both options. They both improve the issue enough. But if I have to choose, I prefer option 2 just a little because:
- it has a higher contrast ratio
- spacing and size add enough clear hierarchy to distinguish the primary and secondary text lines.

But again they both look good to me! Thanks :)

**Comment by cdrini** [comment]
I prefer option 1, I think the contrast ratio is sufficient as per the guidelines--I do want us to avoid "optimizing" for a theoretical maximum here which might not actually benefit our patrons. I also like the extra visual variety/rhythm of the change in colours :) Gives it a bit of depth besides having it communicate visual hierarchy.

**Comment by kmcgonigle** [comment]
To give a potential third option: we could have the colors in Option 1 and add the Spacing from Option 2.  🤔

**Comment by tfmorris** [comment]
Option 3 please. Making the secondary text darker makes it look cramped without increasing the vertical spacing.

**Comment by schu96** [comment]
I'd like to be assigned to this issue, here's what Option 3 looks like with a darker text and increased text spacing

Before:
![image](https://github.com/user-attachments/assets/80ec7801-b055-403f-9c8b-fe947160d96d)

Option 3:
![image](https://github.com/user-attachments/assets/c32f552c-8e9f-44c1-8946-74a76586d9df)

**Comment by mekarpeles** [comment]
Option 3 looks fine to me, if we've verified it satisfies the accessibility checks.

If you're still willing @schu96, feel free to take this one on :)
